libsplinter/src/admin/service/mod.rs

```diff
@@ -221,7 +221,9 @@ impl AdminService {
     /// circuits should be initialized using the service orchestrator. This may not include all
     /// services if they are not supported locally. It is expected that some services will be
     /// started externally.
-    fn restart_services(&self) -> Result<(), ServiceStartError> {
+    ///
+    /// Also creates the requests the connection for members of the circuits and proposals
+    fn restart_circuits(&self) -> Result<(), ServiceStartError> {
```

Comment:
Perhaps this should be `re_initialize_circuits`, as this is doing more than restarting (and re-referencing isn't a restart operation, really).

Reply:
updated